src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/store/commands.py

```diff
@@ -0,0 +1,55 @@
+
+# account trusted id provider operations
+cli_command(__name__, 'datalake store account trustedprovider create', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'create_or_update'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider update', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'update'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider list', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'list_by_account'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider show', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'get'), cf_datalake_store_account_trusted_provider)
+cli_command(__name__, 'datalake store account trustedprovider delete', adls_format_path.format('trusted_id_providers_operations', 'TrustedIdProvidersOperations', 'delete'), cf_datalake_store_account_trusted_provider)
+
+# filesystem operations
+cli_command(__name__, 'datalake store filesystem show', adls_custom_format_path.format('get_adls_item'))
+cli_command(__name__, 'datalake store filesystem list', adls_custom_format_path.format('list_adls_items'))
+cli_command(__name__, 'datalake store filesystem create', adls_custom_format_path.format('create_adls_item'))
+cli_command(__name__, 'datalake store filesystem append', adls_custom_format_path.format('append_adls_item'))
+cli_command(__name__, 'datalake store filesystem delete', adls_custom_format_path.format('remove_adls_item'))
+cli_command(__name__, 'datalake store filesystem upload', adls_custom_format_path.format('upload_to_adls'))
+cli_command(__name__, 'datalake store filesystem download', adls_custom_format_path.format('download_from_adls'))
+cli_command(__name__, 'datalake store filesystem download', adls_custom_format_path.format('download_from_adls'))
+cli_command(__name__, 'datalake store filesystem test', adls_custom_format_path.format('test_adls_item'))
+cli_command(__name__, 'datalake store filesystem preview', adls_custom_format_path.format('preview_adls_item'))
+cli_command(__name__, 'datalake store filesystem join', adls_custom_format_path.format('join_adls_items'))
+cli_command(__name__, 'datalake store filesystem move', adls_custom_format_path.format('move_adls_item'))
+# todo implement set expiry when it is available
+
+# filesystem permission operations
+# todo: implement acl CRUD when available
+cli_command(__name__, 'datalake store filesystem permission set', adls_custom_format_path.format('set_adls_item_permissions'))
+cli_command(__name__, 'datalake store filesystem owner set', adls_custom_format_path.format('set_adls_item_owner'))
```

Comment:
If the `permission` and `owner` subgroups have only a single command, consider moving them "up" one level as hyphenated commands: `store filesystem set-permission/set-owner`.